doc/examples/reconst_dki.py

```diff
@@ -48,99 +52,99 @@
                     & W_{xxzz} & W_{yyzz} & W_{xxyz} & W_{xyyz} & W_{xyzz}
                     & & )\end{matrix}
 
-In the following example we show how to reconstruct your diffusion multi-shell
-datasets using the kurtosis tensor model.
-
-First import Dipy's DKI module:
+In the following example we show how to fit the diffusion kurtosis model on
+diffusion-weighted multi-shell datasets and how to estimate diffusion kurtosis
+based statistics.
 
+First, we import all relevant modules:
 """
 
 import dipy.reconst.dki as dki
+import dipy.reconst.dti as dti
+import matplotlib.pyplot as plt
+import nibabel as nib
+from dipy.data import fetch_cenir_multib
+from dipy.data import read_cenir_multib
+from dipy.segment.mask import median_otsu
+from dipy.denoise.noise_estimate import estimate_sigma
+from dipy.denoise.nlmeans import nlmeans
 
 """
 DKI requires multi-shell data, i.e. data acquired from more than one non-zero
-b-value. Here, we use fetch to download the raw HARDI of a dMRI dataset
-cotaining three non-zero b-values. The size of the dataset is 188 MBytes,
-however you only need to fetch it once.
+b-value. Here, we use fetch to download a multi-shell dataset which parameters
```

Comment:
=> "... a multi-shell dataset with parameters that are similar to the data collected in the Human Connectome Project..."